pinot-controller/src/main/java/org/apache/pinot/controller/ControllerStarter.java

```diff
@@ -211,8 +214,8 @@ public void start() {
   private void setUpHelixController() {
     // Register and connect instance as Helix controller.
     LOGGER.info("Starting Helix controller");
-    _helixControllerManager = HelixSetupUtils
-        .setup(_helixClusterName, _helixZkURL, _instanceId, _isUpdateStateModel, _enableBatchMessageMode);
+    _helixControllerManager = HelixSetupUtils.setup(_helixClusterName, _helixZkURL, _instanceId, _isUpdateStateModel,
```

Comment:
Have you changed your setting for the code format? Seems this part is reformatted back and forth.